package.json

```diff
@@ -34,124 +34,126 @@
   "dependencies": {
     "@hapi/wreck": "^17.0.0",
     "@octokit/graphql": "^4.3.1",
-    "@reach/portal": "^0.9.0",
-    "@reach/router": "^1.3.1",
-    "@reach/tooltip": "^0.9.1",
-    "@types/react": "^16.9.27",
-    "@types/styled-components": "^5.0.1",
+    "@reach/portal": "^0.10.0",
+    "@reach/router": "^1.3.3",
+    "@reach/tooltip": "^0.10.0",
     "classnames": "^2.2.6",
     "color": "^3.1.2",
     "compression": "^1.7.4",
-    "date-fns": "^2.8.1",
+    "date-fns": "^2.11.1",
     "docsearch.js": "^2.6.3",
+    "ease-component": "^1.0.0",
     "express": "^4.17.1",
     "fs-extra": "^9.0.0",
-    "gatsby": "^2.20.2",
-    "gatsby-image": "^2.3.0",
-    "gatsby-link": "^2.3.0",
-    "gatsby-plugin-feed": "^2.4.0",
-    "gatsby-plugin-typescript": "^2.2.5",
-    "gatsby-plugin-webpack-bundle-analyzer": "^1.0.5",
-    "github-markdown-css": "^3.0.1",
+    "gatsby": "^2.20.13",
+    "gatsby-image": "^2.3.1",
+    "gatsby-link": "^2.3.2",
+    "github-markdown-css": "^4.0.0",
+    "iso-url": "^0.4.7",
     "isomorphic-fetch": "^2.2.1",
-    "lodash.fill": "^3.4.0",
     "lodash.includes": "^4.3.0",
     "lodash.kebabcase": "^4.1.1",
     "lodash.startcase": "^4.4.0",
     "lodash.throttle": "^4.1.1",
     "lodash.topairs": "^4.3.0",
```

Comment:
aside: we have tree-shaking so we can simply do:

```
import { get } from 'lodash'
```

Knowing only `get` is included in the bundle. So we don't have to install lodash bit by bit.

Reply:
Unfortunately not =( Tree shaking usually works like a 💩 
There are a lot articles which describe this problem, like this one https://www.azavea.com/blog/2019/03/07/lessons-on-tree-shaking-lodash/
But we can replace it via `import get from lodash/get` and it should work, but maybe it can be done in another PR (or tomorrow if it still won't be merged)